judge/views/tag.py

```diff
@@ -226,5 +229,5 @@ def get_comment_page(self):
     def get_context_data(self, **kwargs):
         context = super(TagProblemDetail, self).get_context_data(**kwargs)
 
-        context['title'] = self.object.name
+        context['title'] = mark_safe(format_html('<a href="{0}">{1}</a>', self.object.link, self.object.name))
```

Comment:
This can lead to the wrong title in the browser tab 
![image](https://user-images.githubusercontent.com/23715841/129432441-0f193311-a7ef-436d-a880-3d3682190372.png)

Please inherit this class with `TitleMixin`  class, and override the `get_title` and `get_content_title` instead